pkg/convert/convert.go

```diff
@@ -76,91 +77,229 @@ func RegionsCsvToResources(resourcesFile string) ([]map[string]interface{}, erro
+						"      size: %s\n"+
+						"      generic_name: %s\n",
+					id, resource[Name].(string), resource[CloudProviderId].(string), resource[CpuCores].(string),
+					resource[Memory].(string), resource[Category].(string), resource[SizePretty].(string),
+					resource[GenericName].(string)))
+				_, err = file.Write(machineTypeString)
+				if err != nil {
+					panic(err)
+				}
+				i++
+			}
+		}
+	} else {
+		if appInterfaceDir[len(appInterfaceDir)-1] == '/' {
+			appInterfaceDir = appInterfaceDir[:len(appInterfaceDir)-1]
+		}
+		appInterfaceConfigmapFile := fmt.Sprintf("%s/resources/services/ocm/cloud-resources.configmap.yaml", appInterfaceDir)
+		file, err := os.Open(appInterfaceConfigmapFile)
+		if err != nil {
+			return err
+		}
 
-	// Machine types
-	i := 0
-	_, err = file.Write([]byte("---\napiVersion: v1\nkind: ConfigMap\nmetadata:\n  " +
-		"annotations:\n    " + "qontract.recycle: \"true\"\n  " + "name: cloud-resources-config\ndata:\n  instance-types.yaml: |\n    instance_types:\n"))
-	if err != nil {
-		panic(err)
-	}
-	for _, resource := range resources {
-		if resource[ResourceType] == "compute.node" {
-			// Create block for this machine type
-			id := helper.AssignStringValue(resource["id"])
-			machineTypeString := []byte(fmt.Sprintf(
-				"    - id: %s\n"+
-					"      name: %s\n"+
-					"      cloud_provider_id: %s\n"+
-					"      cpu_cores: %s\n"+
-					"      memory: %s\n"+
-					"      category: %s\n"+
-					"      size: %s\n"+
-					"      generic_name: %s\n",
-				id, resource[Name].(string), resource[CloudProviderId].(string), resource[CpuCores].(string),
-				resource[Memory].(string), resource[Category].(string), resource[SizePretty].(string),
-				resource[GenericName].(string)))
-			_, err = file.Write(machineTypeString)
-			if err != nil {
-				panic(err)
+		fmt.Println(fmt.Sprintf("Rewriting app-interface configmap (%s) with new data supplied from CSV...\n", appInterfaceConfigmapFile))
+
+		scanner := bufio.NewScanner(file)
+		finalText := ""
+		newConfigmap := ""
+		for i, resource := range resources {
+			if resource[ResourceType] == "compute.node" {
+				if i == 0 {
+					continue
+				}
+
+				id := helper.AssignStringValue(resource["id"])
+
+				data := []byte(fmt.Sprintf(
+					"    - id: %s\n"+
+						"      name: %s\n"+
+						"      cloud_provider_id: %s\n"+
+						"      cpu_cores: %s\n"+
+						"      memory: %s\n"+
+						"      category: %s\n"+
+						"      size: %s\n"+
+						"      generic_name: %s\n",
+					id, resource[Name].(string), resource[CloudProviderId].(string), resource[CpuCores].(string),
+					resource[Memory].(string), resource[Category].(string), resource[SizePretty].(string),
+					resource[GenericName].(string)))
+
+				newConfigmap += string(data)
 			}
-			i++
 		}
+
+		skip := false
+		for scanner.Scan() {
+			text := scanner.Text()
+			if strings.Contains(text, "instance_types:") {
+				finalText += fmt.Sprintf("%s\n", text)
+				finalText += fmt.Sprintf("%s", newConfigmap)
```

Comment:
I think this could be replaced with 
```suggestion
				finalText += fmt.Sprintf("%s\n%s", text, newConfigmap)
```